spec/models/card_spec.rb

```diff
@@ -22,13 +22,13 @@
     end
 
     it "is the correct review_date card" do
-      expect(card.review_date).to eq DateTime.now
+      expect(card.review_date).to eq DateTime.now.utc
     end
   end
 
   context "not valid object" do
     before(:each) do
-      @card1 = Card.create(original_text: 'house', translated_text: 'house', review_date: DateTime.now + 10.day)
+      @card1 = Card.create(original_text: 'house', translated_text: 'house', review_date: DateTime.now.utc + 10.day)
```

Comment:
Line is too long. [116/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.